packages/ripple-address-codec/.eslintrc.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
     'jsdoc/require-jsdoc': 'off',
     'jsdoc/require-param': 'off',
     'tsdoc/syntax': 'off',
+    '@typescript-eslint/no-require-imports': 'off',
```

Comment:
Why was this added?

Reply:
<img width="893" alt="image" src="https://user-images.githubusercontent.com/7017500/208543284-d741f85b-43c0-4450-a34b-c60b8b9d8e3e.png">

Reply:
Is it not possible to use `import * as brorand` syntax for this?

Reply:
No because of the way broand does their export